src/jsx/Converter.js

```diff
@@ -26,15 +26,14 @@ const Converter = (props) => {
   const [modalText, setModalText] = useState({
     mode: 'loading',
     title: {
-      loading: '⭐ Task Started!',
-      success: '🌈 Task Finished!',
-      error: '❌ Task Error!',
+      loading: ' Task in Progress',
+      success: ' Success - Complete',
```

Comment:
Hi @christinerogers, changes here need to be reverted.

Reply:
ok line 30 reverted.